bitbucket-pipelines.yml

```diff
@@ -0,0 +1,91 @@
+# Define the Docker image to use for all steps
+image: 
+  name: aaronmedinaaligent/shopify-cli-base:latest
+  username: $DOCKER_HUB_USERNAME
+  password: $DOCKER_HUB_PASSWORD
+
+definitions:
+  steps:
+    - step: &theme-check
+        name: Theme Check
+        services:
+          - docker
+        caches:
```

Comment:
Would this cache be required in this step?

Reply:
See comments in services part below.